WcaOnRails/app/controllers/api/v0/persons_controller.rb

```diff
@@ -19,6 +19,16 @@ def show
     render json: person_to_json(person)
   end
 
+  def medals
+    person = Person.current.find_by_wca_id!(params[:wca_id])
+    render json: person.medals
+  end
+
+  def records
+    person = Person.current.find_by_wca_id!(params[:wca_id])
+    render json: person.records
+  end
```

Comment:
What's the reason for the two methods above? If they were to be their own endpoints, you'd need to update the routes as well, but I don't think that was the intention ^^

Reply:
Oops! As I actually don't use them, I'll just remove them :)